mlrun/api/crud/artifacts.py

```diff
@@ -110,6 +151,7 @@ def list_artifacts(
             category,
             iter,
             best_iteration,
+            producer_id=tree,
```

Comment:
its confusing that we sometimes call it tree and sometimes producer id.
I know we can't fix all places that do this but I think we should strive to use `tree` at 99% of the places

Reply:
I agree about the confusing part, but don't you think we should strive to use `producer_id` instead of `tree`? 
IMO it is a representation of what it really is